src/queries/get/CacheQueries.ts

```diff
@@ -331,22 +396,85 @@ export async function fetchReadOnlyTerms(
               );
               setSchemeColors(AppSettings.viewColorPool);
             }
-          }
+          } else if (sendTo) Object.assign(sendTo, result);
```

Comment:
What does this line really do? It is not clear whether the result is returned at all.